### DISCUSSION TITLE
Card styling

### DISCUSSION THREAD

**Discussion by ooliver1** [community/community#13284] [Other Feature Feedback, Questions, & Ideas]
Personally, the icons are a bit too small, the description text is a bit too hard to see at a glance

the way the normal `following` feed works better for me and my stupid little eyes

**Comment by sntwandy** [comment]
![github](https://user-images.githubusercontent.com/34795970/159693395-090b5a53-e609-40b0-b62f-c7261f2cad79.png)
Maybe something like this? I put less padding and the letters a little bigger, and yes, it looks a little better and more thinking of those who have bigger screens and it's a little more difficult.

**Reply by ooliver1** [reply]
i was mostly referring to the `starred a repository` and 
![image](https://user-images.githubusercontent.com/34910574/159766996-e96493f0-57aa-4043-b844-47ca0e246095.png)
 icon being quite small - my laptop screen is 15'' iirc

![before](https://cdn.tooty.xyz/raoO)
![after](https://cdn.tooty.xyz/GujM)

the white text is more legible as white, the icon is too small to be useful really
